### ISSUE TITLE
wifi on iyokan

### ISSUE THREAD

**Issue by johanneswilm** [LegacyXperia/local_manifests#150]
On the snapshot from November 1st, I could get Wifi to work by executing /system/bin/wifi-cal.sh . On the version from November 12th runnign this seemingly has no effect -- the wifi stays off, also after restart.

**Comment by ghost** [comment]
Can you try removing the file /data/misc/wifi/wl1271-nvs.bin before you run /system/bin/wifi-cal.sh . i.e.

su
rm /data/misc/wifi/wl1271-nvs.bin
/system/bin/wifi-cal.sh

If that doesn't work post the output of dmesg and logcat.

Did you remember to flash the kernel from the build?

**Comment by mikeNG** [comment]
Always flash the included kernel of each build and follow the exact installation steps

**Comment by johanneswilm** [comment]
Yes sure. I had to wait a few days for a new micro usb cable to arrive in the mail just to make sure that wasn't the cause. The thing is that since I switched from an older cyonagenmod version to LegacyXperia, FlashTool always complains "Device connected with USB debugging off" even when my device shows the little pirate icon and claims "Android debugging enabled." That's why I haven't been able to flash the kernel lately.
